lib/project_types/extension/tasks/get_product.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require "shopify_cli"
+
+module Extension
+  module Tasks
+    class GetProduct < ShopifyCli::Task
+      API_VERSION = "2021-07"
```

Comment:
I'm new - please excuse my ignorance.  I totally support the use of constants for strings that are liable to change in the future, or used in multiple places.  but it's a bit confusing - is this saying that this task only works with this particular API_VERSION?  isn't this version liable to change in the future, and so shouldn't this constant be defined elsewhere, in a more global context?

Reply:
If no API version is passed to the `AdminApi`, it will default to the unstable version of the API. I do not think it is a good idea to default to the unstable version, so I am defining which API version to use here, at the task level. I simply chose the latest release.

The reason I am not doing this at a more global level is because I do not want to overwrite functionality of other tasks that may be using a different version of the API.

You can learn more about API versioning here: https://development.shopify.io/engineering/developing_at_Shopify/apis/api_version/api_version_tldr